### ISSUE TITLE
Can't scrape the Java section

### ISSUE THREAD

**Issue by VishalRohra** [CuriousLearner/GeeksForGeeksScrapper#9]
Indicates: 0 links found

**Comment by CuriousLearner** [comment]
Yes, I confirm. This isn't working for the `Java` category anymore. 

I've taken a quick look and it seems `soup.select('article li')` does not contain anything.
Seems like the DOM has been changed. Have to inspect it more. Though I doubt it if it is done only for one page.

I'll look into this later. Let me know if you want to take this up!

Thanks!
